neo/Prompt/Commands/tests/test_wallet_commands.py

```diff
@@ -93,6 +171,54 @@ def test_wallet_close(self):
 
                 self.assertTrue(res)
 
+    def test_wallet_verbose(self):
+        with patch('neo.Prompt.PromptData.PromptData.Prompt'):
+            # test wallet verbose with no wallet
+            args = ['verbose']
+            res = CommandWallet().execute(args)
```

Comment:
same as previous. This does not test verbose, but fails on a not having an open wallet.

Reply:
This test is specifically testing `test wallet verbose with no wallet`